my_frst_html.html

```diff
@@ -3,9 +3,9 @@
       <head>
         <meta charset="utf-8">		
 	<meta name="description" content="this is my frst html5 page..."
-        <title>My first webpage</title>
+        <title>My first Blog</title>

```

Comment:
haha... baja master branch me ja aapne project ki
click green merge pull request button there to mege my branch patch-1 to
master branch (yours)
remember you will always work on master branch if you r the lead of the
project.

On Fri, May 13, 2016 at 11:12 PM, Harshit bajaj <notifications@github.com>
wrote:

> In my_frst_html.html
> <https://github.com/harshitbajaj/sharedproject/pull/1#discussion_r63222467>
> :
>
> > @@ -3,9 +3,9 @@
> >        <head>
> >          <meta charset="utf-8">		
> >  	<meta name="description" content="this is my frst html5 page..."
> > -        <title>My first webpage</title>
> > +        <title>My first Blog</title>
>
> nice change himanshu ;)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/harshitbajaj/sharedproject/pull/1/files/b0733d19798d42dd6f58012245d5ce85f061a42e#r63222467>
>